src/utils/build-info.ts

```diff
@@ -77,6 +78,17 @@ export const detectFrameworkSettings = async (
   }
 
   if (settings.length > 1) {
+    if (isCI) {
+      log(`Multiple possible ${type} commands found`)
+      throw new Error(
+        `Detected commands for: ${settings
+          .map((setting) => setting.framework.name)
+          .join(
+            ', ',
+          )}. Update your settings to specify which to use. See docs at: https://docs.netlify.com/configure-builds/file-based-configuration/#netlify-dev`,
```

Comment:
We typically refrain from using "see" as it's considered ableist language. I've also created a short link to make the message easier on the eye, and lets us point the link to a different part of our docs in the future without updating the CLI.

```suggestion
          )}. Update your settings to specify which to use. Refer to https://ntl.fyi/dev-monorepo for more information.`,
```